Feature/nyc council

Gi0rgi0s opened the pull request:
NOTE: For some reason, I can't get the latlon location on the latest dev branch. Everything works with hard-coded GPS. In `RepresentativesManager`, uncomment `repLat` and `repLong` to see working example.
1. Refactored `RepresentativesManager` and `RESTUtil` to handle Request objects rather than URL strings
2. Refactored above classes to handle APIs returning `null`
3. Added NycCouncilApi along with NycCouncilGeoUtil for NYC local reps
4. Removed API keys from build
5. Added raw Councilmember data json file to ./res/raw package

Gi0rgi0s committed:
1. Added working NycCouncilApi
2. Added some Geocoder functionality
3. Refactored APIs to generate Request objects rather than URLs
4. Refactored RESTUtil to accomodate #3
5. Added google-play-services-location to dependencies build.gradle for Geocoding

Gi0rgi0s committed:
Merge branch 'develop' into nyc_council

Gi0rgi0s committed:
1. Minor changes to debugging code in RepresentativesManager<BR>2. Activated NycCouncilApi (was commented out)

squoraishee commented on app/src/main/AndroidManifest.xml:
```diff
@@ -19,15 +19,14 @@
             android:screenOrientation="portrait"
             android:windowSoftInputMode="stateAlwaysHidden|adjustPan">
-
+git
```
Remove this from android manifest

Gi0rgi0s committed:
1. Removed stray typing from AndroidManifest.xml
2. Fixed faulty loop in NycCouncilGeoUtil improving robustness
3. Fixed faulty logic in NycCouncilApi for beter error handling
